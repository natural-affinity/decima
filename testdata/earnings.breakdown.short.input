decima 100 -b
