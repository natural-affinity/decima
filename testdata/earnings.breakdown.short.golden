You owe: $10.00

Takehome: $90.00
Earnings: $100.00
 Percent: 10.00%
